packages/devtools_app/lib/src/screens/profiler/panes/controls/profiler_screen_controls.dart

```diff
@@ -111,15 +111,15 @@ class _SecondaryControls extends StatelessWidget {
             icon: Icons.timer,
             label: 'Profile app start up',
             tooltip:
-                'Load all Dart CPU samples that occurred before \n'
-                'the first Flutter frame was drawn (if available)',
+                'This button is deprecated. The new procedure for profiling an'
```

Comment:
Instead of deprecating the button, I think it would be better to remove it entirely (and update the website docs on CPU profiling to call out this flag)